### DISCUSSION TITLE
Pydantic in Constraints

### DISCUSSION THREAD

**Discussion by wolvery** [apache/airflow#41862] [Q&A]
Hi!
After this [PR](https://github.com/apache/airflow/pull/37320), should the Pydantic be in the constraints file?
It should be ignored.

**Comment by potiuk** [comment]
Constraints are "golden" set of dependencies when we install "everything" - but they do not bind you - and constraints just tell "if you install x the version is y". More information on what constraints should be used for (reproducible installation) and what they should not be used for (diverging from constraints and installing your own version of packages) is explained here https://airflow.apache.org/docs/apache-airflow/stable/installation/installing-from-pypi.html#constraints-files
